### ISSUE TITLE
Error while parsing the empty mapping 

### ISSUE THREAD

**Issue by iamshreeram** [mewil/elastic-graphql#20]
Fails to parse when the mapping is empty or without any properties
```
{[http://es.sitepoint.azure.net:9200] dlake*  true true true true}
2021/12/03 15:06:09 built document type for dlake-mgmt-compute in 207.912µs
2021/12/03 15:06:09 built document type for dlake-events-vv in 69.441µs
2021/12/03 15:06:09 built document type for dlake-compute in 437.844µs
2021/12/03 15:06:09 built document type for dlake-org in 90.15µs
2021/12/03 15:06:09 built document type for dlake-slborg in 519.815µs
panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 94 [running]:
main.buildSchemasFromMapping(0xc0000ac680, 0x1f, 0xc00066daa0, 0xc0001662b8, 0xc0000740c0, 0xc0009880a0)
	/go/src/github.com/mewil/sturgeon/schema.go:42 +0xbcc
created by main.buildSchemaFromMappings
	/go/src/github.com/mewil/sturgeon/schema.go:18 +0x145
panic: interface conversion: interface {} is nil, not map[string]interface {}
```

We may need to [verify if the mapping is not empty](https://github.com/mewil/sturgeon/blob/main/schema.go#L41) before fetching the properties.
```
if len(mapping) != 0 {
   ...
} else {
   ...
}
```

Thoughts?

cc: @mewil

**Comment by mewil** [comment]
Yes, you're right that there needs to be a check to determine if the mapping actually contains `properties`. I've addressed this case in [b0e8408](https://github.com/mewil/sturgeon/commit/b0e84088acfae0153d8d35b3dca7744b41ece101).

**Comment by iamshreeram** [comment]
Yes this works. Now, When I'm running the application in my local, the application is stuck (after building document aggregating type) without exposing any graphiql or stdout'ing any error or exception.

```
2021/12/09 00:37:58 built boolean argument type for dlake_cms_tomcat in 7.715906ms
2021/12/09 00:37:58 built document aggregation type for dlake_cms_tomcat in 531.127µs
2021/12/09 00:37:58 built document aggregation type for dlake_smj_inc in 1.470374ms
2021/12/09 00:37:58 built boolean argument type for dlake_svcnow_smj_inc_v2 in 8.299834ms
2021/12/09 00:37:58 built document aggregation type for dlake_smj_major_inc_v2 in 887.458µs

(no response from the application after this)
```

**Comment by mewil** [comment]
There was a small bug in the initial fix I pushed, but it should be resolved by [50b4d16d](https://github.com/mewil/sturgeon/commit/50b4d16dd8ff4b9bde1f2091cd4da94ee0f0f533).

**Comment by iamshreeram** [comment]
That fixed!
